SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -152,3 +152,62 @@ uint64 FSpatialLoadBalancingHandler::GetLatestAuthorityChangeFromHierarchy(const
 
 	return LatestTimestamp;
 }
+
+void FSpatialLoadBalancingHandler::LogMigrationFailure(EActorMigrationResult ActorMigrationResult, AActor* Actor)
+{
+	FString FailureReason;
+
+	switch (ActorMigrationResult)
+	{
+	case EActorMigrationResult::NotAuthoritative:
+		FailureReason = TEXT("does not have authority");
+		break;
+	case EActorMigrationResult::NotReady:
+		if (Actor->GetGameTimeSinceCreation() > 1)
```

Comment:
I think the no-log period should be extended to most errors, because most of them have a chance to be transient states.
The not-ready case catches an actor created locally, but would not catch an actor created remotely which is currently migrating (which would land in the NotAuth case). Same thing for PendingKill, the actor will go away eventually, etc...

Reply:
Sure, done.